libs/custom-paper-handler/src/cli/driver_cli.ts

```diff
@@ -49,65 +44,13 @@ async function flushTransferInGeneric(driver: PaperHandlerDriverInterface) {
   await driver.clearGenericInBuffer();
 }
 
-async function getOnlyPageFromPdf(pdf: Buffer): Promise<ImageData> {
-  const first = await iter(pdfToImages(pdf, { scale: 200 / 72 })).first();
-  assert(first?.pageCount === 1);
-  return first.page;
-}
-
-async function outputSampleBallotPdfWidth() {
-  const page = await getOnlyPageFromPdf(Buffer.from(ballotFixture));
-  console.log(`First page of sample ballot is ${page.width} dots`);
-}
-
 async function scan(driver: PaperHandlerDriverInterface): Promise<void> {
   const dateString = new Date().toISOString();
   const pathOut = join(tmpdir(), `ballot-driver-cli-${dateString}.jpg`);
   console.log('Writing scan to', pathOut);
   await driver.scanAndSave(pathOut);
 }
 
-/**
- * Unsafely prints a ballot from ballot fixtures. Adapted from paper_handler_machine.
- * Precondition: enable-print command has succeeded
- */
-async function printBallot(driver: PaperHandlerDriverInterface): Promise<void> {
```

Comment:
This was the last caller of `../test/fixtures/ballotFixture`. It's a bit of a code smell/trapdoor to have this possibly divergent printing implementation in a debugging tool, and it's rarely used these days, so seemed best to remove.